terraform/modules/application/role_task_execution_policy.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_iam_role_policy" "parameter_access" {
+  name = "${var.prefix}-parameter-access"
+  role = var.ecs_task_execution_role_id
+
+  policy = jsonencode({

```

Comment:
Thanks! I actually had the data object approach originally on this branch, but changed it back to the above as this was what we had pre-this-refactor, and knew it would work for review apps (where there will be several of these policies generated and applied to the same role) 😅. Would be good to chat about the advantages for my awareness separately 👍